tests/main/api-get-systems-label/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: verify that the /systems/<label> api works
+
+systems:
+  # TODO: also test classic/core hybrid systems once they are ready
+  - ubuntu-core-2*
+
+execute: |
+  snap install --edge jq
+  # devmode as the snap does not have snapd-control
+  snap install test-snapd-curl --devmode --edge
+
+  echo "Find the what systems are available"
+  test-snapd-curl.curl -s --unix-socket /run/snapd.socket http://localhost/v2/systems > systems
+  current_label=$(jq -r '.result.systems[0]["label"]' < systems)
```

Comment:
Thanks, I'm not a big fan of `-o pipefail` as it comes with it's own share of potential confusion, see https://mvogt.wordpress.com/2018/07/23/bash-o-pipefail-can-have-pitfailswpitfalls/ - however in this case it would be fine as jq is not greedy (I think) so it might be a worthwhile change :)

Reply:
Ha, nice point about SIGPIPE, I'll keep that in mind!